enterprise-viewer/latest/using/index.md

```diff
@@ -1517,22 +1494,19 @@ Clicking this button will allow the user to split the document into different se
 
 ![Img Txt]({% link enterprise-viewer/images/aev-edit-pages-interface-toolbar14.png %})
 
-Sectioning is an efficient way to reorder large documents. A blue notification will appear in the bottom right corner
-where the user can name the section and select the range of pages for the section. The user must then select the create
-section button. The new section will appear in the left sidebar. For more information on how to section a document,
-see this [section](#section-docs).
+Sectioning is an efficient way to reorder large documents. A blue notification will appear in the bottom right corner where the user can name the section and select the range of pages for the section. The user must then select the create section button. The new section will appear in the left sidebar.
```

Comment:
Is "create section" an actual button? If it is, I suggest using the bold font face to maintain consistent formatting.

Reply:
Good spot - I missed this one! It's a UI element and worth highlighting as it's a link inside a notification: see https://docs.alfresco.com/enterprise-viewer/latest/using/#creating-a-section.